src/mixins/filing-template-mixin.ts

```diff
@@ -106,31 +101,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
    * @returns the correction filing body
    */
   buildCorrectionFiling (isDraft: boolean): CorrectionFilingIF {
-    let parties = this.getPeopleAndRoles
-    let shareClasses = this.getShareClasses
-    let nameTranslations = this.getNameTranslations
-
-    // if filing...
-    if (!isDraft) {
-      // Filter out removed parties and delete "actions" property
-      parties = parties.filter(x => !x.actions?.includes(ActionTypes.REMOVED))
-        .map((x) => { const { actions, ...rest } = x; return rest })
-
-      // Filter out removed classes and delete "action" property
-      shareClasses = shareClasses.filter(x => x.action !== ActionTypes.REMOVED)
-        .map((x) => { const { action, ...rest } = x; return rest })
-
-      // Filter out removed series and delete "action" property
-      for (const [index, share] of shareClasses.entries()) {
-        shareClasses[index].series = share.series?.filter(x => x.action !== ActionTypes.REMOVED)
-          .map((x) => { const { action, ...rest } = x; return rest })
-      }
-
-      nameTranslations = this.prepareNameTranslations()
-    }
-
```

Comment:
All this got moved to a helper method below (esp since this code was duplicated).